exchange/exchange.go

```diff
@@ -936,6 +936,40 @@ func (e *exchange) getBidCacheInfo(bid *pbsOrtbBid, auction *auction) (cacheInfo
 	return
 }
 
+// getAuctionCurrencyRates returns the a currency conversions implementation that will be used in the rest of this
+// auction. Parameter customRates contains, if any, the bidRequest custom currency conversion rates
+// defined in bidRequest.Ext. If usePbsRates flag is set to false, the currency conversions provided by Prebid Server's
+// currency conversion service get discarded in favor of the bidRequest.ext.currency-defined conversion rates.
+func (e *exchange) getAuctionCurrencyRates(customRates *openrtb_ext.ExtRequestCurrency) currency.Conversions {
```

Comment:
Consider renaming `customRates` to `requestRates` to make the source of the information clearer. Thoughts?

Reply:
I agree. Done